src/components/PackTile.js

```diff
@@ -73,95 +73,122 @@ const PackTile = ({
       className={className}
       interactive
       css={css`
+        --tile-image-height: 100px;
+        padding: 1rem;
         overflow: hidden;
-        display: flex;
-        flex-direction: column;
-
-        ${view === VIEWS.LIST &&
+        display: grid;
+        grid-gap: 0.2rem;
+        grid-template-rows: var(--tile-image-height) 3rem 1fr 0.2fr;
+        grid-template-columns: auto;
+        ${!isListView() &&
         css`
-          margin-bottom: 1em;
-          flex-direction: row;
+          @media screen and (max-width: ${IMAGE_BREAKPOINT}) {
+            grid-template-rows: 3rem 4.5fr 0.5fr;
+          }
         `}
 
-        @media screen and (max-width: ${QUICKSTARTS_COLLAPSE_BREAKPOINT}) {
-          flex-direction: row;
-        }
+        ${isListView() &&
+        css`
+          grid-template-columns: 0.5fr 1fr 1fr;
+          grid-template-areas:
+            'image title tag'
+            'image summary tag';
```

Comment:
As we discussed on the campfire, we might be able to get away with simplifying the styles by using grid for both the grid layout and list layout: https://codepen.io/zstix/pen/WNZKrKe?editors=1100